tensorboard/webapp/widgets/line_chart_v2/lib/types.ts

```diff
@@ -31,3 +31,8 @@ export interface Extent {
   x: [number, number];
   y: [number, number];
 }
+
+/**
+ * Flattened array of 2d coordinates: [x0, y0, x1, y1, ..., xn, yn].
+ */
+export type Paths = Float32Array;
```

Comment:
From the comment I can see that it’s a sequence of points. Why is it
called `Paths`? Is it really a `Points`, or a [`Polyline`][pl], or a
`Polygon`, or something else?

[pl]: https://en.wikipedia.org/wiki/Polyline

Reply:
I like "Polyline" so I applied it everywhere. Thanks for the suggestion.